internal/mage/go.mod

```diff
@@ -23,4 +23,6 @@ require (
 // needed to resolve "ambiguous import: found package cloud.google.com/go/compute/metadata in multiple modules"
 replace cloud.google.com/go => cloud.google.com/go v0.100.2
 
+replace dagger.io/dagger => ../../sdk/go
```

Comment:
After reading the dagger [release instructions](https://github.com/dagger/dagger/blob/722a002444954331c459bc4b29182e4e16458a35/RELEASING.md#release-1) I'm realizing that I went a little too far here.